GetServices/handler.ts

```diff
@@ -69,13 +76,23 @@ export function GetServicesHandler(
         }, {})
     );
 
-    return tryCatch(() => allServicesIterator.next(), toCosmosErrorResponse)
+    return tryCatch(
+      () => asyncIteratorToArray(allServicesIterator),
+      toCosmosErrorResponse
+    )
       .fold<IGetServicesHandlerResult>(
         error => ResponseErrorQuery("Cannot get services", error),
-        iteratorResults => {
+        results => {
+          // tslint:disable-next-line: no-inferred-empty-object-type
+          const reducedResults = results.reduce((prev, curr) => {
+            return {
+              ...prev,
+              ...curr
+            };
+          }, {});
```

Comment:
what's the purpose of this code?

Reply:
results' type is `{}[]`. I've used reduce to flatten results into `{}`

Reply:
nevermind, changed implementation in [85a8305](https://github.com/pagopa/io-functions-admin/pull/69/commits/85a8305ddacd25efb495fa696662e5479d9f7740)